### DISCUSSION TITLE
Workspace scroll without workspace buttons in i3bar?

### DISCUSSION THREAD

**Discussion by bancobusto** [i3/i3#5914] [Q&A] [answered]
Hi, first post here. I apologize if the answer to my question has been answered elsewhere. I appreciate any hints you may have with my small issue. 

Inside the bar brackets in my i3 config file I have the command: 

workspace_buttons no

I typically use a number of workspaces at once and unlike many I do not wish to see the workspace numbers. I find them distracting. 

What I learned is that with the buttons removed from the bar, I can no longer scroll over the bar or panel to change workspaces. Often that's fine as I have other custom keybindings to achieve my goals but **_occasionally_**  I miss the feature (though not enough to re-enable the workspace buttons). 

To be clear I'm not looking for a Mod key suggestion, whereas if my hands are near enough the keyboard (as is often the case) I would simply use my keyboard shortcuts. 

Any suggestions? This is certainly not a game changer for me one way or the other and I did look in the manual as best as I could. Thank you to the i3 team for an amazingly clear, well documented piece of software! It has changed the way that I operate and couldn't imagine having it any other way :)

Mike 

Edited to add that I am running i3 version 4.22 on Debian stable.

**Comment by orestisfl** [comment, marked as answer]
Try

```
bar {
  workspace_buttons no
  bindsym button4 workspace prev
  bindsym button5 workspace next
}
```

**Reply by bancobusto** [reply]
Thank you. I was somehow unable to find this post, and add the exact answer that you have given! Despite me thinking that I had 'read the manual' I didn't grasp the concept of mouse scroll / button4 and five. Man, do I love i3. Thank you for taking the time to answer this, so that someone can find the answer in the future. Cheers!

**Reply by orestisfl** [reply]
No worries. Regarding this part:
> I typically use a number of workspaces at once and unlike many I do not wish to see the workspace numbers. I find them distracting.

Maybe the new `workspace_command` can allow you to create workspace buttons according to what works best for you? E.g. you could only display the currently focused workspace. See https://i3wm.org/docs/userguide.html#workspace_command
